stdlib/REPL/src/REPL.jl

```diff
@@ -206,13 +206,28 @@ end
 function display(d::REPLDisplay, mime::MIME"text/plain", x)
     io = outstream(d.repl)
     get(io, :color, false) && write(io, answer_color(d.repl))
+    io = IOContext(io, :interactive => true)
```

Comment:
```suggestion
```

I think we can remove this for now.

(It'd be useful to have this, but I think it's clear to do it in a separate PR that also introduces a proper docstring to `IOContext`)